src/css/inputs.css

```diff
@@ -0,0 +1,53 @@
+@layer utilities {
+    .radio-indicator {
+        @apply border border-solid border-theme-primary-700 dark:border-theme-primary-600 h-5 w-5 absolute top-0 rounded-[10px] transition-smoothEase hover:bg-[#CCEEE0] hover:dark:bg-theme-primary-900 focus:shadow-[0_0_0_2px_#E5FFF4] focus:shadow-[#E5FFF4] active:shadow-[0_0_0_2px_#E5FFF4] after:content-[""] after:absolute after:hidden
+    }
+
+    [type='radio']:checked + .radio-indicator::after {
+        @apply block border border-solid border-theme-primary-700 bg-theme-primary-700 rounded-full dark:border-theme-primary-600 dark:bg-theme-primary-600 w-2 h-2 top-1/2 left-1/2 transform -translate-y-1/2 -translate-x-1/2
+    }
+
+    [type='radio']:disabled:checked + .radio-indicator::after {
+        @apply bg-theme-secondary-300 border-theme-secondary-300 dark:bg-theme-secondary-100 dark:border-theme-secondary-100
+    }
+
+    [type='radio']:disabled + .radio-indicator {
+        @apply pointer-events-none border border-solid border-theme-secondary-300 bg-theme-secondary-100 dark:bg-black
+    }
+
+    .toggle-switch-slider {
+        @apply absolute cursor-pointer top-0 left-0 right-0 bottom-0 bg-theme-secondary-200 dark:bg-theme-secondary-600 duration-[0.4s] rounded-xl z-[1] hover:bg-theme-primary-700 dark:hover:bg-theme-primary-650 group-hover:bg-[#CCEEE0] dark:group-hover:bg-theme-primary-900
+    }
+
+    .toggle-switch-slider::before {
+        @apply absolute content-[""] h-4 w-4 left-[0.2rem] bottom-[0.15rem] bg-white duration-[0.4s] rounded-full fill-white drop-shadow-[0_1px_3px_rgba(16,24,40,0.10)]
```

Comment:
didn't notice this prior, but 0.2rem (and 0.15rem) can cause issues as they will result in partial pixels, so it may look ok on one screen but not on the other 🤔 better if we refactor this as the same time to be proper full pixels